addons-l10n/en/editor-devtools.json

```diff
@@ -28,15 +28,15 @@
   "editor-devtools/send-top-bottom": "Send to Top, Send to Bottom",
   "editor-devtools/send-top-bottom-desc": "Right click a costume and 2 new menu items are present. These can be used to send the clicked sprite to the top or bottom of the list of costumes for fast re-ordering.",
   "editor-devtools/youtube": "YouTube tutorials",
-  "editor-devtools/orphaned": "{count, plural, one {Griffpatch: I found 1 orphaned reporter block. Shall I delete it for you?} other {Griffpatch: I found # orphaned reporter blocks. Shall I delete them for you?} }",
-  "editor-devtools/unused-var": "{count, plural, one {Griffpatch: I found 1 unused local variable. Shall I delete it for you? Here it is:\n} other {Griffpatch: I found # unused local variables. Shall I delete them for you? Here they are:\n} }",
+  "editor-devtools/orphaned": "{count, plural, one {Developer tools: I found 1 orphaned reporter block. Shall I delete it for you?} other {Developer tools: I found # orphaned reporter blocks. Shall I delete them for you?} }",
+  "editor-devtools/unused-var": "{count, plural, one {Developer tools: I found 1 unused local variable. Shall I delete it for you? Here it is:\n} other {Developer tools: I found # unused local variables. Shall I delete them for you? Here they are:\n} }",
```

Comment:
I think it would also make sense to change "I found ..." to "... was/were found".